src/main/java/reposense/parser/AuthorConfigCsvParser.java

```diff
@@ -105,7 +104,6 @@ private static AuthorConfiguration findMatchingAuthorConfiguration(
      */
     private static void setEmails(AuthorConfiguration config, Author author, List<String> emails) {
         author.setEmails(new ArrayList<>(emails));
-        config.addAuthorEmailsAndAliasesMapEntry(author, author.getEmails());
```

Comment:
may i get the reason on why these `config.addAuthor..` lines are removed?

Reply:
All these are redundant as all of these are already set by the parent function before these functions are called. And these functions are called only one by one parent function.  Therefore, removed them.

Reply:
By "parent function", are you referring to `processLine` ? 
I think the removed lines are not redundant.
In `processLine`, we indeed call `config.addAuthor` which technically does store the author's emails and aliases via the `addAuthorEmailsAndAliasesMapEntry` method.

However, the complication is that the `processLine` method calls `config.addAuthor`  **before** the `Author` object is initialised with any emails and aliases.  This means that no emails and aliases are stored in the `AuthorConfiguration` when `config.addAuthor` is called - because the `Author` object is not fully initialised yet.

Hence, the removed lines 108, 124, 130 are not redundant. 


**Unrelated to this PR** -  the flow of events mentioned above should be refactored in my view.
We should initialise the `Author` instances before calling `config.addAuthor`.

Reply:
Alright! I just checked this. I agree. We cannot remove this as of now. Will open an issue for the refactor.